packages/store-ui/src/organisms/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,47 @@
+import type { ChangeEvent } from 'react'
+import React, { useRef, useState } from 'react'
+
+import type { PopoverProps } from '../../atoms/Popover'
+import Popover from '../../atoms/Popover'
+import type { SearchInputProps } from '../../molecules/SearchInput'
+import SearchInput from '../../molecules/SearchInput'
+
+export type SearchBarProps = Pick<
+  SearchInputProps,
+  'onSubmit' | 'onChange' | 'placeholder' | 'testId'
+> &
+  Pick<PopoverProps, 'children'>
+
+const SearchBar = ({
+  onSubmit,
+  onChange,
+  children,
+  placeholder,
+  testId = 'store-search-bar',
+}: SearchBarProps) => {
+  const ref = useRef(null)
+  const [value, setValue] = useState('')
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setValue(event.target.value)
```

Comment:
What do you think about debouncing this setState to prevent too much re-render or use uncontrolled input?